addons/mail/static/tests/tours/user_modify_own_profile_tour.js

```diff
@@ -1,4 +1,5 @@
 import { registry } from "@web/core/registry";
+import { contains } from "@web/../tests/utils";
```

Comment:
`web` are legacy, we re-moved them to `mail` for HOOT. Is that a different version that only works with tour?

Reply:
Indeed, this version is used in tours. Mail version is not available in the tour bundles.